LKExp-Firmware-Disco-Pelion_via_ethernet/mbed_app.json

```diff
@@ -0,0 +1,667 @@
+            "target.restrict_size"                      : "0xE7C00",
+            "update-client.bootloader-details"          : "0x8005d30",
+            "update-client.application-details"         : "(MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_BASE_ADDRESS + MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-address"             : "(MBED_ROM_START + (MBED_ROM_SIZE/2))",
+            "update-client.storage-size"                : "((MBED_ROM_SIZE/2) - MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-locations"           : 1,
+            "update-client.storage-page"                : 1,
+            "update-client.firmware-header-version"     : "2",
+            "mbed-cloud-client.update-storage"          : "ARM_UCP_FLASHIAP",
+            "storage.storage_type"                      : "TDB_INTERNAL",
+            "storage_tdb_internal.internal_base_address": "(MBED_ROM_START + 32*1024)",
+            "storage_tdb_internal.internal_size"        : "(32*2*1024)"
+        },
+        "CY8CPROTO_062_4343W": {
+            "target.macros_add"                         : ["MBEDTLS_USER_CONFIG_FILE=\"mbedTLSConfig_mbedOS.h\"","ARM_UC_DELTAPAAL_WRITE_BUF_SIZE=512"],
+            "target.network-default-interface-type"     : "WIFI",
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_remove"                  : ["CM0P_SLEEP"],
+            "target.bootloader_img"                     : "bootloader/mbed-bootloader-cy8cproto-062-4343w-ext-kvstore-qspi-v4.1.3.bin",
+            "bootloader-size"                           : "0x00020000",
+            "target.header_offset"                      : "0x00020000",
+            "target.app_offset"                         : "0x00020400",
+            "mbed-cloud-client.update-storage"          : "ARM_UCP_FLASHIAP_BLOCKDEVICE",
+            "update-client.bootloader-details"          : "0x10010B9C",
+            "update-client.application-details"         : "0x10020000",
+            "update-client.storage-address"             : "0x00000000",
+            "update-client.storage-size"                : "0x1C0000",
+            "update-client.storage-page"                : 512,
+            "storage.storage_type"                      : "TDB_EXTERNAL",
+            "storage_tdb_external.blockdevice"          : "QSPIF",
+            "storage_tdb_external.internal_base_address": "0x101E0000",
+            "storage_tdb_external.rbp_internal_size"    : "0x10000",
+            "storage_tdb_external.external_base_address": "0x00200000",
+            "storage_tdb_external.external_size"        : "4*0x40000",
+            "target.bootloader_supported"               : true,
+            "target.mbed_rom_start"                     : "0x10000000",
+            "target.mbed_rom_size"                      : "0x00200000",
+            "target.sectors"                            : [[268435456, 512]]
+        }
```

Comment:
Ca fait beaucoup de targets 😅 Peut être ne garder que celle qui nous concerne

Reply:
Oui c'est prévu, il faudra faire le tri avec celle qui nous serons utile pour faire les transitions de configuration. Pour l'instant on est au début.

Reply:
Done in 15d45dd